pages/operations/uninstall.md

```diff
@@ -28,3 +39,24 @@ plural destroy
 {% callout severity="danger" %}
 Only do this if you're absolutely sure you want to bring down all associated resources with this repository.
 {% /callout %}
+
+## Terraform Destroy
+
+To tear down the current cluster but leave installation references as pointers to the helm/terraform, cd into `bootstrap/terraform` and run:
+
+```
+terraform destroy
+```
+
+
+## Start from Zero
+
+To remove all installation references in our API, run:
+
+```
+plural repos reset
+```
+
+This command does not interact with any infrastructure, but removes references to all installations. This will also reset any association with a specific cloud provider.
+
+To tear down a cluster and fully start over, run `plural destroy` and then `plural repos reset`.
```

Comment:
@michaeljguarino does a user need to do plural repos reset after plural destroy to remove those installations, or is that cleaned up now?